packages/adminrouter/extra/src/nginx.agent.conf

```diff
@@ -58,5 +58,13 @@ http {
             include common/proxy-headers.conf;
             proxy_pass http://metrics/;
         }
+
+        # Group: System
+        # Description: DC/OS version (unauthenticated)
+        location /dcos-metadata/dcos-version.json {
```

Comment:
We don't have any shared locations, just shared upstreams. Are there other common locations or is this the first?